engine-tests/src/tests/eth_connector.rs

```diff
@@ -1363,3 +1404,11 @@ fn test_ft_transfer_wrong_u128_json_type() {
         "Expected failure as number type can't be parsed to u128",
     );
 }
+
+/// Bytes for a NEAR smart contract implementing `ft_on_transfer`
+fn ft_receiver_bytes() -> Vec<u8> {
```

Comment:
```suggestion
fn dummy_ft_receiver_bytes() -> Vec<u8> {
```

Reply:
Done in https://github.com/aurora-is-near/aurora-engine/pull/330/commits/a9c6a2908ce489696d9c7a6f7ac8a11572e3153f